src/utils/credential.js

```diff
@@ -0,0 +1,72 @@
+import * as jwt from './jwt';
+import { decode } from 'jsonwebtoken';
```

Comment:
I think that was only required by the node lib though. In this case , the credential file is required by both the node lib and the browser lib